packages/calcite-components/src/components/time-picker/time-picker.tsx

```diff
@@ -309,6 +312,11 @@ export class TimePicker
     }
   }
 
+  @Listen("mousedown")
```

Comment:
For compatibility, can you use `pointerdown` instead? Applies to all `mouse`-related code.

Reply:
It needs to use the `@Listen` decorator for it to work correctly, for some reason.  When I inline the pointerdown handler on the Host element, the `focusHandler` method is getting an out-of-date value for `pointerActivated`, resulting in the flash of focus styling.  I can go into more detail if you'd like to see it in action.